src/components/02_molecules/navigation/Navigation.tsx

```diff
@@ -109,12 +109,15 @@ export const Navigation: React.FC<NavigationProps> = () => {
             description={firstDoc.title}
             size="default"
             compact
+            className={s.navLinkImage}
           >
             <ExportedImage
               src={firstDoc.image}
               alt={firstDoc.title}
               fill
               placeholder="empty"
+              sizes="(max-width: 759px) 100vw, 50vw"
+              unoptimized
```

Comment:
Hey @MarcelKnobelsdorf, why did you add the property `unoptimized` to the exported image here?